cloud/profitbricks/profitbricks.py

```diff
@@ -0,0 +1,632 @@
+    required: false
+    default: us/las
+    choices: [ "us/las", "us/lasdev", "de/fra", "de/fkb" ]
+  assign_public_ip:
+    description:
+      - This will assign the machine to the public LAN. If no LAN exists with public Internet access it is created.
+    required: false
+    default: false
+  lan:
+    description:
+      - The ID of the LAN you wish to add the servers to.
+    required: false
+    default: 1
+  subscription_user:
+    description:
+      - The ProfitBricks username. Overrides the PB_SUBSCRIPTION_ID environement variable.
+    required: false
+    default: null
+  subscription_password:
+    description:
+      - THe ProfitBricks password. Overrides the PB_PASSWORD environement variable.
+    required: false
+    default: null
+  wait:
+    description:
+      - wait for the instance to be in state 'running' before returning
+    required: false
+    default: "yes"
+    choices: [ "yes", "no" ]
+    aliases: []
+  wait_timeout:
+    description:
+      - how long before wait gives up, in seconds
+    default: 600
+    aliases: []
+  remove_boot_volume:
+    description:
+      - remove the bootVolume of the virtual machine you're destroying.
+    required: false
+    default: "yes"
+    choices: ["yes", "no"]
+  state:
+    description:
+      - create or terminate instances
+    required: false
+    default: 'present'
+    aliases: []
+
+requirements: [ "profitbricks" ]
+author: Matt Baldwin (baldwin@stackpointcloud.com)
+'''
+
+EXAMPLES = '''
+
+# Note: These examples do not set authentication details, see the AWS Guide for details.
+
+# Provisioning example. This will create three servers and enumerate their names. 
+
+- profitbricks:
+    datacenter: Tardis One
+    name: web%02d.stackpointcloud.com
+    cores: 4
+    ram: 2048
+    volume_size: 50
+    image: a3eae284-a2fe-11e4-b187-5f1f641608c8
+    location: us/las
+    count: 3
+    assign_public_ip: true
+
+# Removing Virtual machines
+
+- profitbricks:
+    datacenter: Tardis One
+    instance_ids:
+      - 'web001.stackpointcloud.com'
+      - 'web002.stackpointcloud.com'
+      - 'web003.stackpointcloud.com'
+    wait_timeout: 500
+    state: absent
+
+# Starting Virtual Machines.
+
+- profitbricks:
+    datacenter: Tardis One
+    instance_ids:
+      - 'web001.stackpointcloud.com'
+      - 'web002.stackpointcloud.com'
+      - 'web003.stackpointcloud.com'
+    wait_timeout: 500
+    state: running
+
+# Stopping Virtual Machines
+
+- profitbricks:
+    datacenter: Tardis One
+    instance_ids:
+      - 'web001.stackpointcloud.com'
+      - 'web002.stackpointcloud.com'
+      - 'web003.stackpointcloud.com'
+    wait_timeout: 500
+    state: stopped
+
+'''
+
+import re
+import uuid
+import time
+import json
+import sys
+
+try:
+    from profitbricks.client import ProfitBricksService, Volume, Server, Datacenter, NIC, LAN
+except ImportError:
+    print "failed=True msg='profitbricks required for this module'"
+    sys.exit(1)
```

Comment:
modules should not print nor exist, use a variable to capture the import failure and then use fail_json to give out a the message, look at ec2 or openstack plugins as an example